l2geth/cmd/utils/flags.go

```diff
@@ -887,34 +886,16 @@ var (
 		Value:  eth.DefaultConfig.Rollup.MaxCallDataSize,
 		EnvVar: "ROLLUP_MAX_CALLDATA_SIZE",
 	}
-	RollupDataPriceFlag = BigFlag{
```

Comment:
Is it correct that this PR simply makes it so that we get these values from the GasPriceOracle instead of from input values? If so, how are the values in the GasPriceOracle initialized?

Reply:
Yes, they are initialized in the `SyncService`
https://github.com/ethereum-optimism/optimism/blob/0c18e1903f8a33a607f782c0081a8fb5071b71ef/l2geth/rollup/sync_service.go#L205-L206

Reply:
They can also be updated via the `PrivateRollupAPI` in the `rollup_personal` RPC namespace

https://github.com/ethereum-optimism/optimism/blob/0c18e1903f8a33a607f782c0081a8fb5071b71ef/l2geth/internal/ethapi/api.go#L2016

https://github.com/ethereum-optimism/optimism/blob/0c18e1903f8a33a607f782c0081a8fb5071b71ef/l2geth/internal/ethapi/api.go#L2021